Makefile

```diff
@@ -97,7 +97,7 @@ upgrade: check_pins	## update the requirements/*.txt files with the latest packa
 	$(PIP_COMPILE) -o requirements/doc.txt requirements/doc.in
 	$(PIP_COMPILE) -o requirements/test.txt requirements/test.in
 	$(PIP_COMPILE) -o requirements/dev.txt requirements/dev.in
-	$(PIP_COMPILE) -o requirements/travis.txt requirements/travis.in
+	$(PIP_COMPILE) -o requirements/github.txt requirements/github.in
```

Comment:
Just a suggestion, In all other services we use `github.in` for github dependencies and have switched the name `travis.in` to `ci.in` & `ci.txt`.